kotlin/README.md

```
@@ -122,26 +102,127 @@ mapOf(
+- The library handles the PKCE flow internally.
+- User-trust based credential download supported through onCheckIssuerTrust callback.
+- This method is the recommended way to request credential using credential offer and will replace `requestCredentialByCredentialOffer` in future releases.
+
+###### Parameters
+
+| Name                    | Type                     | Required | Default Value | Description                                                                                                                                                                               |
+|-------------------------|--------------------------|----------|---------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| credentialOffer         | String                   | Yes      | N/A           | Credential offer as embedded JSON or `credential_offer_uri`                                                                                                                               |
+| clientMetadata          | ClientMetadata           | Yes      | N/A           | Contains client ID and redirect URI                                                                                                                                                       |
+| getTxCode               | TxCodeCallback           | No       | N/A           | Optional callback function for TX Code (for Pre-Auth flows)                                                                                                                               |
+| authorizations          | List<Authorization>      | Yes      | N/A           | Callback functions list to handle authorization and return the resultant authorization response (for Authorization flows) [click [here](#authorizations) for details about authorization] |
```

markdownlint-cli2 (0.21.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)